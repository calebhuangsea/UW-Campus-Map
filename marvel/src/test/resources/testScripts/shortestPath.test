# Find a path with where there exist 2 paths
# between start node and end node with different distance

LoadGraph shortestPath shortestPath.csv

FindPath shortestPath n1 n3