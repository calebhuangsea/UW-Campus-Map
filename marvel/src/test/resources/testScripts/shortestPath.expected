# Find a path with where there exist 2 paths
# between start node and end node with different distance

loaded graph shortestPath

path from n1 to n3:
n1 to n3 via a3